README.md

```diff
@@ -1 +1,3 @@
-# layout
+layout
+=========
+See the [page](https://www.polymer-project.org/docs/polymer/layout-attrs.html) for more information
```

Comment:
Can you make the link text more informational?

    See the [layout attributes documentation](...) for more information.